### PR TITLE
TOF: trending for hit multiplicity

### PR DESCRIPTION
**Pull request by njacazio**

### CODE REVIEW

**Comment by njacazio** [comment]
@ercolessi

File: Modules/TOF/include/TOF/TaskDigits.h [hunk 1]
```diff
@@ -88,28 +92,22 @@ class TaskDigits final : public TaskInterface
...
-  std::shared_ptr<TH2F> mTOFmacropadCTTM = nullptr;         /// Map of hit macropads according to CTTM numbering
-  std::shared_ptr<TH2F> mTOFmacropadDeltaPhiTime = nullptr; /// #Deltat vs #Delta#Phi of hit macropads
-  std::shared_ptr<TH2I> mBXVsCttmBit = nullptr;             /// BX ID in TOF matching window vs trg channel
-  std::shared_ptr<TH2F> mTimeVsCttmBit = nullptr;           /// TOF raw time vs trg channel
-  std::shared_ptr<TH2F> mTOFRawHitMap24 = nullptr;          /// TOF average raw hits/channel map (1 bin = 1 FEA = 24 channels)
-  std::shared_ptr<TH2I> mHitMultiVsDDL = nullptr;           /// TOF raw hit multiplicity per event vs DDL
-  std::shared_ptr<TH1I> mNfiredMacropad = nullptr;          /// Number of fired TOF macropads per event
-  std::shared_ptr<TProfile2D> mOrbitDDL = nullptr;          /// Orbits per crate
-  std::shared_ptr<TH1I> mROWSize = nullptr;                 ///Readout window size
+  // std::shared_ptr<TH1F> mTOFRawsLTMHits = nullptr;          /// LTMs OR signals
```
Comment (Barthelemy) [inline Modules/TOF/include/TOF/TaskDigits.h:95]: We usually encourage people to remove their code rather than commenting out. Not very important.

**Review by Barthelemy — APPROVED** [review]
Ok for me

File: Modules/TOF/include/TOF/TaskDigits.h (hunk 1 above)
Comment (njacazio) [reply Modules/TOF/include/TOF/TaskDigits.h:95]: Indeed this was more to keep it as a bookmark

**Comment by Barthelemy** [comment]
unrelated error, merging

**Pull request merged**